docs/kubernetes/getstarted.md

```diff
@@ -0,0 +1,604 @@
+# Kubernetes cluster provisioning, deploiement and examples on Threefold Grid
+
+In this guide we will walk you through the provisioning of a full-blown kubernetes cluster
+on the TF grid.
+
+We will then see how to connect to it and interact using kubectl on our local machine.
+
+Finally we will go through some examples use cases to grasp the features offered by the cluster.
+
+# TL;DR
+
+If you quickly want to setup a kubernetes cluster and start to play with it, you can launch the startup script.
+
+Prerequisite:
+
+- linux based OS
+- [kubectl](https://kubernetes.io/fr/docs/tasks/tools/install-kubectl/)
+- github account with [ssh key linked to your account](https://help.github.com/en/enterprise/2.17/user/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+
+```
+$ chmod +x startup.sh
+$ ./startup.sh zgorizzo69 "173.40" 1 2
+```
+
+This will build tfuser provision a network
+
+# Provisioning and Deploiement of the cluster
+
+Our CLI tool tfuser makes it easy to interact with the TFGrid and provision a kubernetes cluster.
```

Comment:
tfuser is only a developer tool and only works on `devnet`. Let's be very explicit about that.